### PR TITLE
Feature blossom

### PR DESCRIPTION
**Pull request by itsthejoker**
## Description:

Finally: the Blossom integration is complete. Does what it says on the box.

## Checklist:

- [x] Code Quality
- [x] Pep-8
- [ ] Tests (if applicable)
- [x] Success Criteria Met
- [x] Inline Documentation
- [ ] Wiki Documentation (if applicable)

### CODE REVIEW

File: tor_ocr/core/initialize.py [hunk 1]
```diff
@@ -1,267 +1,48 @@
...
-    config.subreddits_domain_filter_bypass = get_wiki_page(
-        'subreddits/domain-filter-bypass', config
-    ).split('\r\n')
-    config.subreddits_domain_filter_bypass = clean_list(
-        config.subreddits_domain_filter_bypass
+def get_blossom_connection():
+    return BlossomAPI(
+        email=os.getenv("BLOSSOM_EMAIL"),
+        password=os.getenv("BLOSSOM_PASSWORD"),
+        api_key=os.getenv("BLOSSOM_API_KEY"),
```
Comment (MaxVanDeursen) [inline tor_ocr/core/initialize.py:35]: Also requires base url

File: tor_ocr/main.py [hunk 2]
```diff
@@ -1,284 +1,146 @@
...
-        }
-        return error
-
-    json_result = decode_image_from_url(image_url)
+b_api = BlossomAPI(
+    email=os.environ.get('TOR_OCR_EMAIL'),
+    password=os.environ.get('TOR_OCR_PASSWORD'),
+    api_key=os.environ.get('TOR_OCR_BLOSSOM_API_KEY'),
+    api_base_url=os.environ.get('TOR_OCR_BLOSSOM_API_BASE_URL'),
+    login_url=os.environ.get('TOR_OCR_BLOSSOM_API_LOGIN_URL')
```
Comment (MaxVanDeursen) [inline tor_ocr/main.py]: The login URL should be removed.

File: tor_ocr/main.py [hunk 3]
```diff
@@ -1,284 +1,146 @@
...
 
-    if not result:
-        logging.info('Result was none! Skipping!')
-        # we don't want orphan entries
-        config.redis.delete(new_post)
-        return
-
-    tor_post_id = config.redis.get(new_post).decode('utf-8')
+    for post in new_posts:
+        # there should be two transcriptions: one from the volunteer
```
Comment (MaxVanDeursen) [inline tor_ocr/main.py]: Is this necessarily the case? If a post has just been submitted, there is not necessarily a transcription by the volunteer right?

**Review by MaxVanDeursen — CHANGES_REQUESTED** [review]
Some small requests, but other than that nice job!

File: tor_ocr/main.py (hunk 2 above)
Comment (itsthejoker) [reply tor_ocr/main.py]: Ooooooh this got pulled when I merged with the previous code. Crap.

File: tor_ocr/main.py (hunk 3 above)
Comment (itsthejoker) [reply tor_ocr/main.py]: Excellent point, that is true

**Review by MaxVanDeursen — DISMISSED** [review]

**Pull request merged**